modules/observability-workload-monitoring.adoc

```diff
@@ -67,8 +67,8 @@ spec:
 $ oc apply -f monitoringServiceMonitor.yaml
 ----
 
-Prometheus scrapes the path `/metrics` by default, however you can define a custom path. 
-It is up to the vendor of the application to expose this endpoint for scraping, with metrics that they deem relevant.
+Prometheus scrapes the `/metrics` path by default. However, you can define a custom path. 
+The vendor of the application must decide whether to expose the endpoint for scraping, with metrics that they deem relevant.
```

Comment:
I'm not familiar with how this works, but could the vendor of the application just be the user? 

Could you remove this line? Prometheus scrapes the /metrics endpoint by default, but you can define a custom path.  Whatever metrics are scraped are determined by how the user application is configured? Just slightly confused by who the vendor of the application is or what that means

Reply:
@rfisher001 Could you please take a look?